tests/Application/Actions/Donations/CreateTest.php

```diff
@@ -1072,4 +1071,39 @@ private static function someCampaignFunding(): CampaignFunding
 
         return $campaignFunding;
     }
+
+    private static function somePaymentIntentArgs(): array
```

Comment:
Went with a class var and set in `setup()` since const values and var defaults cannot be expressions

Reply:
Ah I hadn't spotted that it contained a call to `getenv('APP_ENV')`. Not sure about static class var since that makes it look like one test case may influence the next run to run, although it wouldn't in this case since getenv will return the same every time.